site/src/jsMain/kotlin/com/zenmo/web/zenmo/components/widgets/InlineLinks.kt

```diff
@@ -0,0 +1,57 @@
+package com.zenmo.web.zenmo.components.widgets
+
+import androidx.compose.runtime.Composable
+import com.varabyte.kobweb.compose.css.FontStyle
+import com.varabyte.kobweb.compose.css.TextDecorationLine
+import com.varabyte.kobweb.compose.ui.Modifier
+import com.varabyte.kobweb.compose.ui.modifiers.color
+import com.varabyte.kobweb.compose.ui.modifiers.fontStyle
+import com.varabyte.kobweb.compose.ui.modifiers.textDecorationLine
+import com.varabyte.kobweb.silk.components.navigation.Link
+import com.varabyte.kobweb.silk.components.navigation.LinkStyle
+import com.varabyte.kobweb.silk.style.addVariant
+import com.zenmo.web.zenmo.theme.SitePalette
+import org.jetbrains.compose.web.dom.Div
+
+@Composable
+fun InlineLinks(
+    destinationUrl: String,
+    enLinkText: String,
+    nlLinkText: String,
+    enTextBeforeLink: String,
+    nlTextBeforeLink: String,
+    enTextAfterLink: String? = null,
+    nlTextAfterLink: String? = null,
```

Comment:
This before and after text seems overengineered to me.

The editor can just put a LangText element before and after himself.